### DISCUSSION TITLE
Incompatible method getRelationCountHash  after update to 8.17.2

### DISCUSSION THREAD

**Discussion by webard** [GeneaLabs/laravel-model-caching#496] [General]
```
ErrorException (Declaration of Kalnoy\Nestedset\BaseRelation::getRelationCountHash() should be compatible with Illuminate\Database\Eloquent\Relations\Relation::getRelationCountHash($incrementJoinCount = true)) thrown while looking for class Kalnoy\Nestedset\DescendantsRelation.
```

after update to the lates version 8.17.2

**Comment by mikebronner** [comment]
Hi @webard, thanks for submitting this issue! :) I will try to see what's going on as soon as possible, no promises though, as we are in the middle of moving. I would welcome PRs with failing tests and fixes, if you find yourself in a position to submit them. :) Thanks!
